sdk/include/reactos/wine/debug.h

```diff
@@ -93,7 +93,7 @@ struct __wine_debug_channel
        __WINE_DBG_LOG
 
 #define __WINE_DBG_LOG(args...) \
-    ros_dbg_log( __dbcl, __dbch, __RELFILE__, __FUNCTION__, __LINE__, args); } } while(0)
+    ros_dbg_log( __dbcl, __dbch, __RELFILE__, __LINE__, __FUNCTION__, args); } } while(0)
```

Comment:
Why are you swapping these parameters by the way? That just adds noise to the diff.

Reply:
For an aesthetical purpose only: you go from the file (containing many functions), to the function (coded on many lines), to the particular line.